app/src/main/java/eu/kanade/tachiyomi/data/backup/BackupCreatorJob.kt

```diff
@@ -17,7 +18,7 @@ class BackupCreatorJob(private val context: Context, workerParams: WorkerParamet
 
     override fun doWork(): Result {
         val preferences = Injekt.get<PreferencesHelper>()
-        val backupManager = BackupManager(context)
+        val backupManager = if (preferences.isFullBackup().get()) FullBackupManager(context) else BackupManager(context)
```

Comment:
It probably makes more sense to have a setting to toggle the additional creation of a legacy online backup file on top of the default offline backup file.